src/Stratis.Bitcoin.IntegrationTests/SmartContracts/SmartContractWalletTests.cs

```diff
@@ -89,7 +88,8 @@ public void SendAndReceiveCorrectly()
                 Assert.Null(scReceiver.FullNode.WalletManager().GetSpendableTransactionsInWallet(WalletName).First().Transaction.BlockHeight);
 
                 // generate two new blocks do the trx is confirmed
-                scSender.GenerateBlockManually(new List<Transaction>(new[] { scSender.FullNode.Network.CreateTransaction(trx.ToBytes()) }));
+                scSender.AddToStratisMempool(scSender.FullNode.Network.CreateTransaction(trx.ToBytes()));
```

Comment:
Is this to follow up on what we talked about in the previous PR? (add to mempool then mine)

Reply:
Hi Jeremy, yes that is correct 👍

Reply:
I was able to change some of the tests to use this.